cluster-up/container.sh

```diff
@@ -2,16 +2,24 @@
 
 set -e
 
+if [ -z "$KUBEVIRTCI_PATH" ]; then
+    KUBEVIRTCI_PATH="$(
+        cd "$(dirname "$BASH_SOURCE[0]")/"
+        echo "$(pwd)/"
+    )"
+fi
+
+source ${KUBEVIRTCI_PATH}/hack/common.sh
+
 if [[ $KUBEVIRT_PROVIDER =~ (ocp|okd).* ]]; then
-    POSTFIX="cluster"
-    CONTAINER=$(docker ps | grep kubevirt | grep "${KUBEVIRT_PROVIDER}-${POSTFIX}" | awk '{print $1}')
-    if [ -z $CONTAINER ]; then
-        echo "container was not found"
-        exit 0
-    fi
-    docker exec $CONTAINER bash -c "if ! grep "/root/install/auth/kubeconfig" ~/.bashrc > /dev/null; \
-                                    then echo export KUBECONFIG=/root/install/auth/kubeconfig >> ~/.bashrc; fi"
-    docker exec -it $CONTAINER bash
+    CONTAINER=${provider_prefix}-cluster
```

Comment:
This looks like you moved some logic into `hack/common.sh`? I don't see the change here, so I suppose it has been moved already in another commit?
EDIT: ah, I see, you moved it into `ssh.sh`.

Reply:
I have slim down this scripts.